package.json

```diff
@@ -43,7 +44,8 @@
     "eslint": "^6.8.0",
     "prettier": "1.19.1",
     "eslint-config-prettier": "6.9.0",
-    "eslint-plugin-prettier": "3.1.2"
+    "eslint-plugin-prettier": "3.1.2",
+    "eslint-plugin-import": "^2.20.1"
```

Comment:
Nothing to change for now, but I think we need to review how our dependencies are setup.
E.g. everything should really be a production dependency to avoid us manually needing to import some stuff.

Reply:
I agree, just followed how the rest of the project was setup atm.